sagemaker-debugger/tensorflow2/tensorflow2_keras_custom_container/tf2-keras-custom-container.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
<p>if user will run this it wont use custom container , rather default container? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
correct. if user doesn't mention an image name, it will use SM DLC